config/app.js.bak

```diff
@@ -0,0 +1,41 @@
+var createError = require("http-errors");
```

Comment:
Since we use git to keep track of files, we don't need to create .bak file. Please delete.

Reply:
ok got it!